public/data/khan-droid.json

```diff
@@ -0,0 +1,17 @@
+{
+    "name": "Khalid Khan",
+    "bio": "Upcoming Full stack developer | Open source contributor",
+    "avatar": "https://github.com/khan-droid.png",
+    "links": [
+      {
+        "name": "Follow me on GitHub",
+        "url": "http://github.com/khan-droid",
+        "icon": "github"
+      },
+      {
+        "name": "Follow me on Twitter",
+        "url": "http://twitter.com/khalid_khan111",
+        "icon": "twitter"
+      }
+    ]
+  }
```

Comment:
Needs a blank line at the end, see the red mark by GitHub on line 17

Reply:
```suggestion
}

```